BtConnectorLib/btconnectorlib2/src/main/java/org/thaliproject/p2p/btconnectorlib/BluetoothBase.java

```diff
@@ -35,53 +31,58 @@ public BluetoothBase(Context Context, BluetoothStatusChanged handler) {
 
         //bluetooth = new BluetoothAdapter(this);
         bluetooth = BluetoothAdapter.getDefaultAdapter();
-        if(bluetooth != null) {
+        if (bluetooth != null) {
             blueAddress = bluetooth.getAddress();
```

Comment:
Please remove blueAddress and instead please make getAddress into an accessor that calls bluetooth.getAddress() directly.